File: cpp/bc1074.cpp
#include <iostream>

using namespace std;

int main()
{
	int in;
	string res = "";
	
	cin >> in;
	
	for(int i = 0; i < in; i++)
	{
		int val;
		
		cin >> val;
		
		if(val == 0) res += "NULL";
		else
		{
			if(val % 2 == 0) res += "EVEN";
			else res += "ODD";
			if(val > 0) res += " POSITIVE";
			else res += " NEGATIVE";
		}
		
		cout << res << endl;
		res = "";
	}
}

File: python/bc1074.py
iN = int(input())

for i in range(0, iN):
    res = ""
    val = int(input())

    if(val == 0): res = "NULL"
    else:
        res += "EVEN" if val % 2 == 0 else "ODD"
        res += " POSITIVE" if val > 0 else " NEGATIVE"
    print(res)